app/controllers/api/job_log_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  class JobLogController < ApplicationController
+    respond_to :json
+    
+    def index
+      joblog = CompletedJob.last_ten_jobs(params[:course_id],params[:github_repo_id])
```

Comment:
I'm wondering whether these params exist in this use case... maybe they are always nil.

If so, it won't impact correctness of the running code, but it might confuse future developers.  If we can figure that out now, it might help "future us". :-)

Reply:
Consider replacing with

```
      joblog = CompletedJob.last_ten_jobs(nil,nil)
```

In a separate follow up issue.